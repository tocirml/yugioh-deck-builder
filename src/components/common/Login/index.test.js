import React from 'react';
import { Login } from './index';

import { shallow } from 'enzyme';

describe('rendering', () => {
  it('renders without crashing', () => {
    shallow(<Login />);
  });
});

describe('snapshots', () => {
  it('Anvorgesa snapshot', () => {
    const tree = shallow(<Login />);
    expect(tree).toMatchSnapshot();
  });
});
