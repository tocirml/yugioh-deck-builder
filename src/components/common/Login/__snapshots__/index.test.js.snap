// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots Anvorgesa snapshot 1`] = `
<div
  className="login "
  data-testid="login"
>
  <a
    className="log-in"
    href="#"
  >
    Login
  </a>
  <a
    className="sign-up"
    href="#"
  >
    Sign Up
  </a>
</div>
`;
